semantic_model_generator/validate/context_length.py

```diff
@@ -1,20 +1,42 @@
+import copy
+
 from loguru import logger
 
-_MODEL_CONTEXT_LENGTH_TOKENS = 6000  # We use 6k, with 1.2k for prompt, so that we can reserve ~1k for response tokens.
+from semantic_model_generator.data_processing.proto_utils import (
+    proto_to_yaml,
+    yaml_to_semantic_model,
+)
+
+_MODEL_CONTEXT_LENGTH_TOKENS = 6500  # We use 6.5k, with 1.2k for instructions, so that we can reserve 500 for response tokens (average is 300).
 _MODEL_CONTEXT_INSTR_TOKEN = 20  # buffer for instr tokens
 
 
-def validate_context_length(yaml_str: str, throw_error: bool = True) -> None:
+def validate_context_length(
+    yaml_str: str, throw_error: bool = True, remove_vqr: bool = True
```

Comment:
put throw_error default as False?

Reply:
and `remove_vqr` probably don't need to be a param... can't think of cases that we don't want to remove vqr

Reply:
done